packages/fxa-settings/src/components/Ready/en.ftl

```diff
@@ -1,8 +1,9 @@
 ## Ready component
 
-ready-confirmation = Your password has been reset
+fxa-reset-password-complete-header = Your password has been reset
 # This is a string that tells the user they can use whatever service prompted them to reset their password
 # Variables:
-# $serviceName (String) - the service which caused the user to reset their password
+# { $serviceName } represents a product name (e.g., Mozilla VPN) that will be passed in as a variable
```

Comment:
Small nit, I'm not super sure we need `that will be passed in as a variable`, since that's implied.

Reply:
I copied and pasted this from another comment in an l10n file, since I thought the matching comments might be a useful hint to localizers that this is the same context, but happy to change if you feel strongly about it!

Reply:
Definitely don't feel strongly about it!

Reply:
From a localizer point of view, either comment should be fine.